src/EditorFeatures/Core.Wpf/QuickInfo/OnTheFlyDocsView.xaml.cs

```diff
@@ -111,6 +127,34 @@ public OnTheFlyDocsView(ITextView textView, IViewElementFactoryService viewEleme
                     _responseControl,
                 ]));
 
+        // Locates the "upgrade now" link in the localized text, surrounded by square brackets.
+        var quotaExceededMatch = Regex.Match(
+                EditorFeaturesResources.Chat_limit_reached_upgrade_now_or_wait_for_the_limit_to_reset,
+                @"^(.*)\[(.*)\](.*)$");
```

Comment:
this does not seem ok.  There is nothing that even guarantees that the localized strings will be surrounded by square brackets.  

it's unclear to me why we need to grab out this text to begin with.  Why can't 'upgrade now' be something we just get localized?